daemon/main.go

```diff
@@ -90,6 +91,7 @@ var (
 	v6ServicePrefix       string
 	k8sAPIServer          string
 	k8sKubeConfigPath     string
+	dockerEndpoint        string
```

Comment:
Don't you want to move this to `pkg/containerd`?

Reply:
This is just a temporary storage place to pass it into the `Init()` function later.